### ISSUE TITLE
The UrlInput icon button doesn't have any accessible name

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#1854]
The UrlInput added in https://github.com/WordPress/gutenberg/commit/4dc462c9a8c73a4c7a2e4fe4a6b6fe1cb73e3b52 doesn't have any text or aria-label, is completely "empty" and there's nothing assistive technologies can announce.
It needs an `aria-label` attribute with a meaningful text.

Edit: as per the title, this issue is about the UrlInput _icon button_.

**Comment by afercia** [comment]
The UrlInput has been significantly refactored and this doesn't apply any longer.
